tests/CMakeLists.txt

```diff
@@ -146,12 +143,21 @@ if(AVIF_GTEST)
     add_avif_gtest(avify4mtest)
 
     if(NOT AVIF_CODEC_AOM OR NOT AVIF_CODEC_AOM_ENCODE)
-        # These tests are supported with aom being the encoder. If the aom encoder is unavailable,
-        # these tests are disabled because other codecs may not implement all the necessary features.
-        # For example, SVT-AV1 requires 4:2:0 images with even dimensions of at least 64x64 px.
         set_tests_properties(
-            avifallocationtest avifgridapitest avifincrtest aviflosslesstest avifmetadatatest avifpixitest PROPERTIES DISABLED
-                                                                                                                      True
+            # These tests are supported with aom being the encoder. If the aom encoder is unavailable,
+            # these tests are disabled because other codecs may not implement all the necessary features.
+            # For example, SVT-AV1 requires 4:2:0 images with even dimensions of at least 64x64 px.
+            avifallocationtest
+            avifgridapitest
+            avifincrtest
+            aviflosslesstest
+            avifmetadatatest
+            avifpixitest
+            # rav1e and svt do not support encoder changes (potentially different bit depth between
+            # primary item and bit depth extension).
+            avif16bittest
+            avifaltrtest
```

Comment:
Could you explain why we did not need to disable avif16bittest and avifaltrtest before and need to disable them now?